src/onegov/org/models/search.py

```diff
@@ -86,6 +86,12 @@ def load_batch_results(self) -> list['Hit']:
         This methods is a wrapper around `batch.load()`, which returns the
         actual search results form the query. """
 
+        def get_sort_key(event: Event) -> tuple[bool, float]:
+            if event.latest_occurrence and hasattr(event.latest_occurrence,
+                                                   'start'):
```

Comment:
```suggestion
            if event.latest_occurrence:
```
This should be good enough, `start` is defined on the base class and not nullable.